tests/test_signal_analog_detectors.py

```diff
@@ -9,6 +9,7 @@
 from signal_analog.flow import Assign, Data, Detect, Program, Ref, When
 from signal_analog.charts import TimeSeriesChart
 from signal_analog.detectors import EmailNotification, PagerDutyNotification, \
+                                    BigPandaNotificataion, \
```

Comment:
There's a typo here that prevents the tests from running successfully :(
https://travis-ci.org/Nike-Inc/signal_analog/jobs/589151209

Reply:
It should be better now!

Reply:
Thanks!